File: app/src/main/java/com/qwe7002/telegram_rc/root_kit/shell.java
package com.qwe7002.telegram_rc.root_kit;

import com.topjohnwu.superuser.Shell;

public class shell {
    public static boolean runShellCommand(String command) {
        Shell.Result result = Shell.cmd("su "+command).exec();
        return result.isSuccess();
    }

    public static boolean checkRoot() {
        return Shell.getShell().isRoot();
    }

}

File: app/src/main/java/com/qwe7002/telegram_rc/root_kit/Shell.kt
package com.qwe7002.telegram_rc.root_kit

import com.topjohnwu.superuser.Shell

object Shell {
    fun runShellCommand(command: String): Boolean {
        val result = Shell.cmd("su $command").exec()
        return result.isSuccess
    }

    @JvmStatic
    fun checkRoot(): Boolean {
        return Shell.getShell().isRoot
    }
}
